eve/methods/common.py

```diff
@@ -702,6 +703,27 @@ def resolve_resource_projection(document, resource, req=None):
         if projection.get(auth_field):
             del projection[auth_field]
 
+    # Support system user only visible fields.
```

Comment:
can we reuse this code as it's duplicated in another method?

Reply:
Good point. Will do that.